backend/.env.sample

```diff
@@ -1,6 +1,7 @@
-APP_PORT=5000
+APP_PORT=8080
```

Comment:
no need to change the sample : I guess the port change is only relevant for your computer ;)